Source/WebCore/style/StyleBuilderCustom.h

```diff
@@ -1562,10 +1562,11 @@ inline void BuilderCustom::applyValueContent(BuilderState& builderState, CSSValu
             // Register the fact that the attribute value affects the style.
             builderState.registerContentAttribute(attr.localName());
         } else if (item.isCounter()) {
-            // FIXME: counter-style: we probably want to review this for custom counter-style.
             auto& counter = downcast<CSSCounterValue>(item);
-            auto listStyle = fromCSSValueID<ListStyleType::Type>(counter.listStyle());
-            builderState.style().setContent(makeUnique<CounterContent>(counter.identifier(), listStyle, counter.separator()), didSet);
+            ListStyleType listStyleType { ListStyleType::Type::Decimal, nullAtom() };
```

Comment:
This should be `{ ListStyleType::Type::CounterStyle, "decimal"_s }` eventually, but it can be changed later on.

Reply:
Actually, I've realized this should be set at the parser level to a CSSValueDecimal like before, and not here. I'm changing it. Then in it will correctly be converted to the right ListStyleType